python-avd/pyavd/_eos_designs/structured_config/network_services/router_bgp.py

```diff
@@ -303,7 +307,7 @@ def _update_router_bgp_vrf_mlag_neighbor_cfg(self: AvdStructuredConfigNetworkSer
         """
         In-place update MLAG neighbor part of structured config for *one* VRF under router_bgp.vrfs
         """
-        if not self._mlag_ibgp_peering_redistribute(vrf, tenant):
+        if not self._mlag_ibgp_peering_redistribute(vrf, tenant) and get(vrf, "redistribute_connected", default=True):
```

Comment:
Would be best to have `self._mlag_ibgp_peering_redistribute` looking for `get(vrf, "redistribute_connected", default=True)` and setting the value based on this (move the and in the self._mlag...)

Reply:
Fixed.